src/FSLibrary/MaxTPSTest.fs

```diff
@@ -203,7 +244,6 @@ let maxTPSTest (context: MissionContext) (baseLoadGen: LoadGen) (setupCfg: LoadG
                                   txrate = middle }
 
                         // On smaller networks, run loadgen on all nodes to better balance the overhead of load generation
```

Comment:
Nit: Remove old comment

Reply:
So the comment is still technically accurate, since I didn't change which nodes generate load. That being said, I think we probably won't need watcher nodes anymore in this mission. I think we can change this later though, and remove the comment then.

Reply:
The comment itself is fine, but you copied and pasted the code and comment for splitting load on all nodes to line 149. I think you forgot to delete the comment here when you were moving it

Reply:
ah yes, sorry, you're right. fixed.